src/common/middleware/crashlytics.js

```diff
@@ -0,0 +1,53 @@
+import { setCustomKey } from "../crashlytics";
+
+// Helper function to safely stringify values
+const safeStringify = (value) => {
+  if (value === null || value === undefined) {
+    return "";
+  }
+  if (typeof value === "boolean" || typeof value === "number") {
+    return value.toString();
+  }
+  if (typeof value === "string") {
+    return value;
+  }
+  if (Array.isArray(value) || typeof value === "object") {
```

Comment:
We don't need the `Array.isArray(value)` condition here, because the typeof Array is also `object` in javascript. It's a little weird, but it would work for arrays too.
 ``` javascript
if (typeof value === "object") {
   JSON.stringify(value)
}
  ```

Reply:
fixed